package.json

```diff
@@ -30,18 +30,18 @@
     "yaml": "^1.10.0"
   },
   "devDependencies": {
-    "@types/jest": "23.3.14",
+    "@types/jest": "29.1.0",
     "@types/node": "18.7.23",
-    "jest": "23.6.0",
-    "jest-snapshot-serializer-raw": "1.2.0",
+    "jest": "29.1.1",
+    "jest-snapshot-serializer-raw": "^1.2.0",
```

Comment:
Did you missed this or any reason not pinned?

Reply:
missed, thank you